### PR TITLE
Add note regarding location of '/___graphql'

### PR DESCRIPTION
**Pull request by abeaclark**

### CODE REVIEW

File: docs/docs/creating-and-modifying-pages.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ modify pages created in core or plugins or to create client-only pages.
 
 To see what pages are being created by your code or plugins, you can query
 for page information while developing in Graph*i*QL. Paste the following
-query in the Graph*i*QL IDE for your site.
+query in the Graph*i*QL IDE for your site. The graphql API is accessible at `/__graphql`.
```
Comment (KyleAMathews) [inline docs/docs/creating-and-modifying-pages.md]: Good call!

I'd assume most people would want to use the IDE not directly query the endpoint.

Perhaps change the language to: "The Graph*i*QL IDE is available when running your sites development server at `HOST:PORT/__graphql` e.g. `localhost:8000/__graphql`."

**Comment by KyleAMathews** [comment]
Pushed a tweak to the copy, thanks!

**Comment by abeaclark** [comment]
Thanks man. Sorry I didn't make it back to this yet. 

Loving Gatsby these days! Great work!

**Comment by KyleAMathews** [comment]
No worries, easy for me to make tweak. Looking forward to more great PRs from ya!

**Comment by jlengstorf** [comment]
Hiya @abeaclark! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**